src/main/java/org/apache/sling/feature/maven/mojos/AggregateFeaturesMojo.java

```diff
@@ -65,6 +68,15 @@
     @Override
     public void execute() throws MojoExecutionException {
         checkPreconditions();
+
```

Comment:
I think this doesn't work - it's totally legitimate to use the aggregate mojo more than once in a project, especially when you use profiles with different aggregations

Reply:
Thanks for reviewing and I see your point.  I left a comment on the JIRA ticket for your consideration.